[2024-05-22T13:51:14.206+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-05-22T13:51:14.245+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: weather_dag_parallel.group_a.upload_S3_to_postgres manual__2024-05-22T13:50:32.868472+00:00 [queued]>
[2024-05-22T13:51:14.260+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: weather_dag_parallel.group_a.upload_S3_to_postgres manual__2024-05-22T13:50:32.868472+00:00 [queued]>
[2024-05-22T13:51:14.261+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 3
[2024-05-22T13:51:14.413+0000] {taskinstance.py:2330} INFO - Executing <Task(PostgresOperator): group_a.upload_S3_to_postgres> on 2024-05-22 13:50:32.868472+00:00
[2024-05-22T13:51:14.438+0000] {standard_task_runner.py:63} INFO - Started process 44770 to run task
[2024-05-22T13:51:14.461+0000] {standard_task_runner.py:90} INFO - Running: ['airflow', 'tasks', 'run', 'weather_dag_parallel', 'group_a.upload_S3_to_postgres', 'manual__2024-05-22T13:50:32.868472+00:00', '--job-id', '65', '--raw', '--subdir', 'DAGS_FOLDER/weatherapi_dag.py', '--cfg-path', '/tmp/tmpohfevoq1']
[2024-05-22T13:51:14.468+0000] {standard_task_runner.py:91} INFO - Job 65: Subtask group_a.upload_S3_to_postgres
[2024-05-22T13:51:14.698+0000] {task_command.py:426} INFO - Running <TaskInstance: weather_dag_parallel.group_a.upload_S3_to_postgres manual__2024-05-22T13:50:32.868472+00:00 [running]> on host ubuntu2204
[2024-05-22T13:51:15.294+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_EMAIL='test@gmail.com' AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='weather_dag_parallel' AIRFLOW_CTX_TASK_ID='group_a.upload_S3_to_postgres' AIRFLOW_CTX_EXECUTION_DATE='2024-05-22T13:50:32.868472+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-05-22T13:50:32.868472+00:00'
[2024-05-22T13:51:15.295+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-05-22T13:51:15.334+0000] {sql.py:276} INFO - Executing:  SELECT aws_s3.table_import_from_s3('city_look_up', '', '(format csv, DELIMITER '','', HEADER true)', 'mybucketfordeveloping', 'us_city.csv', 'eu-north-1'); 
[2024-05-22T13:51:15.345+0000] {base.py:84} INFO - Using connection ID '***_conn' for task execution.
[2024-05-22T13:51:15.379+0000] {base.py:84} INFO - Using connection ID '***_conn' for task execution.
[2024-05-22T13:51:15.919+0000] {sql.py:486} INFO - Running statement:  SELECT aws_s3.table_import_from_s3('city_look_up', '', '(format csv, DELIMITER '','', HEADER true)', 'mybucketfordeveloping', 'us_city.csv', 'eu-north-1'); , parameters: None
[2024-05-22T13:51:22.139+0000] {sql.py:495} INFO - Rows affected: 1
[2024-05-22T13:51:22.227+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-05-22T13:51:22.367+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=weather_dag_parallel, task_id=group_a.upload_S3_to_***, run_id=manual__2024-05-22T13:50:32.868472+00:00, execution_date=20240522T135032, start_date=20240522T135114, end_date=20240522T135122
[2024-05-22T13:51:22.476+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-05-22T13:51:22.522+0000] {taskinstance.py:3498} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-05-22T13:51:22.525+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
